interfaces/cython/cantera/_cantera.pxd

```diff
@@ -120,6 +136,8 @@ cdef extern from "cantera/thermo/Species.h" namespace "Cantera":
         Composition composition
         double charge
         double size
+        void getParameters(CxxAnyMap&) except +translate_exception
+        CxxAnyMap input
```

Comment:
C++ signatures are not consistent: some use 
```C++
CxxAnyMap input
```
while others use
```C++
CxxAnyMap& input()
```
I believe using the latter consistently is easier to maintain long-term. Performance should be the same as those functions will get inlined by the compiler?

Reply:
Please correct me if I'm getting this wrong, but I think the variable is only public in cases like the `Species` class that are mostly just structs where all of the member data is already `public`. So the consistency is with the parent class, not across implementations of the `input` member variable / property.

Reply:
This is mostly a stylistic matter ... per code styling guide, *For most classes, member variables should not be public.*. As `input` is newly introduced, it may make sense to just use the setter methods and default to having `m_input` protected. That way, it's always accessed using a getter method. There shouldn't be a difference in performance ...

Reply:
I take your point, but while looking through this PR for cases to fix, I realized that the only new instance of this is for the `SpeciesThermoInterpType` class, where it is actually implemented as getter (with the underlying member variable `m_input`). The other cases that you see here are just making existing member variables accessible within Cython.

Reply:
It’s admittedly a pretty minor point, so it’s ok to leave as is ... depending on how the other comment is addressed (`getParameters`), it may not be used by itself very often.

Reply:
Marking this as resolved as this was pre-existing.